packages/qwik-nx/src/generators/setup-tailwind/files/.vscode/settings.json__tmpl__

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Just thinking, why do we need this at all? What if devs use only webstorm?

Reply:
removed it.